Libraries/UtilLib/src/main/java/com.artardev/util/numeric/NumberUtil.java

```diff
@@ -112,7 +112,7 @@ public static int factorial(int n)
     {
         int result = 1;
 
-        for (int i = 2; i <= n; ++i)
+        for (long i = 2; i <= n; ++i)
```

Comment:
That change wasn't complete, I should have changed the return value of the method and result's type as long too, so that it could calculate the factorial of bigger numbers. Int is very limited.

Reply:
Makes sense. Factorials can get pretty large